src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-multilingual.sch

```diff
@@ -173,6 +173,11 @@
   <sch:pattern>
     <sch:title>$loc/strings/Metadata</sch:title>
 
+    <sch:rule context="/gmd:MD_Metadata">
```

Comment:
I'm not sure why this is multilingual check?  It is not checking values based on multilingual.  I think it should be in  common.

Reply:
I have moved this logic into the common validation